ohmec.css

```diff
@@ -58,13 +58,47 @@ h1 {
 }
 #map {
   width: 95%;
-  height: 80%;
+  height: 75%;
```

Comment:
This was changed from 80% to 75% in order to get the three input forms to show at the bottom of the HTML screen upon opening